### PR TITLE
Backmerge from appium:2.0

### PR DESCRIPTION
**Pull request by raghavgarg1996**
Implemented run app in background funtionality
Updated driver class method "putAppInBackground"
Validated by implmenting test in vodqa.feature file

### CODE REVIEW

File: src/main/java/com/znsio/teswiz/runner/Driver.java [hunk 1]
```diff
@@ -323,9 +323,13 @@ public void selectNotificationFromNotificationDrawer(By selectNotificationLocato
     }
 
     public void putAppInBackground(int duration) {
-        throw new NotImplementedException("To be migrated to appium 2.0");
-        // todo - implement for appium2.0
-//        ((AppiumDriver) driver).runAppInBackground(Duration.ofSeconds(duration));
+        String platform = String.valueOf(Runner.getPlatform());
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/Driver.java:326]: Why converting to string. You can compare the enum value directly

File: src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java [hunk 2]
```diff
@@ -129,4 +130,18 @@ public VodqaScreen swipeByPassingPercentageAttributes(int atPercentScreenHeight,
...
+    @Override
+    public VodqaScreen putAppInTheBackground(int time) {
+        driver.putAppInBackground(time);
+        visually.checkWindow(SCREEN_NAME, "Home screen after putting app in background");
+        return this;
+    }
+
+    public boolean validateAppWorkInBackground() {
+        LOGGER.info("Validating current app package to know app work in background");
+        String getPackageDetails = ((AndroidDriver) driver.getInnerDriver()).getCapabilities().getCapability("appPackage").toString();
```
Comment (anandbagmar) [inline src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java:144]: how does this comparison ensure the app is in the background?

File: src/test/java/com/znsio/teswiz/screen/vodqa/VodqaScreen.java [hunk 3]
```diff
@@ -39,4 +39,6 @@ public static VodqaScreen get() {
     public abstract boolean isSwipeSuccessful(String tileNumber);
     public abstract VodqaScreen swipeRight();
     public abstract VodqaScreen swipeByPassingPercentageAttributes(int atPercentScreenHeight, int fromPercentScreenWidth, int toPercentScreenWidth);
+    public abstract VodqaScreen putAppInTheBackground(int time);
+    public abstract boolean validateAppWorkInBackground();
```
Comment (anandbagmar) [inline src/test/java/com/znsio/teswiz/screen/vodqa/VodqaScreen.java:43]: Screen will not do any validation

File: src/main/java/com/znsio/teswiz/runner/Driver.java (hunk 1 above)
Comment (raghavgarg1996) [reply src/main/java/com/znsio/teswiz/runner/Driver.java:326]: done

File: src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java (hunk 2 above)
Comment (raghavgarg1996) [reply src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java]: done

File: src/test/java/com/znsio/teswiz/screen/vodqa/VodqaScreen.java (hunk 3 above)
Comment (raghavgarg1996) [reply src/test/java/com/znsio/teswiz/screen/vodqa/VodqaScreen.java]: done

**Pull request closed without merging**